.cspell.json

```diff
@@ -7,12 +7,11 @@
     "minmax",
     "openmct",
     "datasources",
-    "recieved",
-    "evalute",
+    "received",
+    "evaluate",
```

Comment:
we can just remove these altogether, no? since they should be included in one of the dictionaries we're using for cspell